Source/PowerMeter/PowerMeter.cs

```diff
@@ -0,0 +1,134 @@
+﻿using NationalInstruments.ModularInstruments.Interop;
+
+namespace NationalInstruments.ReferenceDesignLibraries
+{
+    public static class PowerMeter
+    {
+        #region Type Definition  
+        /// <summary>
+        /// Defines common settings for power meter.
+        /// </summary>
+        public struct CommonConfiguration
+        {
+            /// <summary>
+            /// Specifies the name of the channel you want to configure.
+            /// </summary>
+            public string Channel_Name;
+
+            /// <summary>
+            /// Specifies the measurement mode to which sensor should be switched.
+            /// </summary>
+            public int Measurement_Mode;
+
+            /// <summary>
+            /// Specifies the units NI-568x uses to return measured values. NI-568x sets the NI568X_ATTR_UNITS attribute to this value.
+            /// </summary>
+            public int Units;
+
+            /// <summary>
+            /// Specifies the trigger source.
+            /// </summary>
+            public int Trigger_Source;
+
+            /// <summary>
+            /// Specifies the polarity of the internal trigger slope.
+            /// </summary>
+            public int Slope;
+
+            /// <summary>
+            /// Specifies the trigger level for the measurement signal.
+            /// </summary>
+            public double Trigger_Level;
+
+            /// <summary>
+            /// Specifies the filter function of a sensor on or off.
+            /// </summary>
+            public bool Averaging;
```

Comment:
The variable and its description are not clear. See further comments below.

Reply:
I have updated the comments, it refers to whether auto-averaging mode is enabled or not.